apps/api/src/app/gene/gene.controller.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Controller,
+  Get,
+  Param,
+  Query,
+} from '@nestjs/common';
+import { GeneService } from './gene.service';
+import {ModuleRef} from "@nestjs/core";
+
+@Controller('genes')
+export class GeneController {
+  constructor(private readonly geneService: GeneService) {}
+
+  @Get()
+  async getHugoGenes(@Query("query") query: string) {
```

Comment:
I probably would prefer a more descriptive method name `searchGenes` or `findGenes`

Reply:
changed getHugoGenes to searchGenes